contrib/ucx-dev

```diff
@@ -0,0 +1,43 @@
+#%Module1.0#####################################################################
```

Comment:
maybe the module could be generated from .in file by configure?
so user would not have to provide path
then it could be part of the build directory

Reply:
added `contrib/ucx-dev.in` generated by configure, not sure if we want to install it, what build dir did you mean?

much simpler:
`module load ./contrib/ucx-dev` and `module unload ucx-dev`

Reply:
will it work in case of out-of-tree build?
```
make build-devel
cd build-devel && ../contrib/configure-devel
make
module load contrib/ucx-dev
```

Reply:
yes ucx-dev.in to ucx-dev is done per configure and stored in the out of tree build directory